flashlight/memory/managers/DefaultMemoryManager.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include <algorithm>
 #include <cstdlib>
+#include <iterator> // needed on MSVC to use back_insertor
```

Comment:
nit: we can remove the comment - we should have `<iterator>` regardless of the platform 👍

Reply:
done